src/egon/data/datasets/heat_demand_timeseries/HTS.py

```diff
@@ -1721,6 +1721,49 @@ def CTS_demand_scale(aggregation_level):
     return CTS_district, CTS_grid, CTS_zensus
 
 
+def store_national_profiles(
+    residential_demand_grid,
+    CTS_demand_grid,
+    residential_demand_dist,
+    CTS_demand_dist,
+):
+    for scenario in CTS_demand_grid.scenario.unique():
+        national_demand = pd.DataFrame(
+            columns=["residential rural", "services rural", "urban central"],
+            index=pd.date_range(
+                datetime(2011, 1, 1, 0), periods=8760, freq="H"
+            ),
+        )
+
+        national_demand["residential rural"] = (
+            residential_demand_grid[
+                residential_demand_grid.scenario == scenario
+            ]
+            .drop("scenario", axis="columns")
+            .sum()
+            .values
+        )
+        national_demand["services rural"] = (
+            CTS_demand_grid[CTS_demand_grid.scenario == scenario]
+            .sum(numeric_only=True)
+            .values
+        )
+        national_demand["urban central"] = (
+            residential_demand_dist[
+                residential_demand_dist.scenario == scenario
+            ]
+            .drop("scenario", axis="columns")
+            .sum()
+            .values
+            + CTS_demand_dist[CTS_demand_dist.scenario == scenario]
+            .drop("scenario", axis="columns")
+            .sum()
+            .values
+        )
+
+        national_demand.to_csv(f"heat_demand_timeseries_DE_{scenario}.csv")
```

Comment:
@ClaraBuettner In case you're in the working dir root, it'd be great to use some subdir for the exported files.